### ISSUE TITLE
Make rgba function be able to handle four arguments

### ISSUE THREAD

**Issue by chrishumphrey1985** [wtho/bulma-css-vars#29]
I'm using versions:
`    "buefy": "^0.8.4",
    "bulma": "^0.8.0",
    "bulma-css-vars": "^0.5.5",`

When running the 'bulma-css-vars' script, I receive the following error - Please could you asssist? : 

```
Executing task: npm run update-bulma-colors <

 test-styles@0.1.0 update-bulma-colors C:\Code\StyleTest\test-styles
bulma-css-vars

(node:17312) UnhandledPromiseRejectionWarning: Error: Only 2 arguments allowed, but 4 were passed.
   ┌──> node_modules\buefy\src\scss\components\_carousel.scss
133│         box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.25);
   │                                  ^^^^^^^^^^^^^^^^^^^ invocation
   ╵
   ┌──> node_modules\bulma-css-vars\functions.sass
51 │ @function rgba($col, $alpha)
   │           ━━━━━━━━━━━━━━━━━━ declaration
   ╵
  node_modules\buefy\src\scss\components\_carousel.scss 133:34  rgba()
  node_modules\buefy\src\scss\components\_carousel.scss 133:34  @import
  node_modules\buefy\src\scss\buefy.scss 4:9                    @import
  src\styles\_main.scss 88:9                                    root stylesheet
    at Object._newRenderError (C:\Code\StyleTest\test-styles\node_modules\sass\sass.dart.js:13472:19)
    at Object._wrapException (C:\Code\StyleTest\test-styles\node_modules\sass\sass.dart.js:13315:16)
    at StaticClosure._renderSync (C:\Code\StyleTest\test-styles\node_modules\sass\sass.dart.js:13290:18)
    at Object.Primitives_applyFunction (C:\Code\StyleTest\test-styles\node_modules\sass\sass.dart.js:1085:30)
    at Object.Function_apply (C:\Code\StyleTest\test-styles\node_modules\sass\sass.dart.js:6003:16)
    at _callDartFunctionFast (C:\Code\StyleTest\test-styles\node_modules\sass\sass.dart.js:7672:16)
    at Object.renderSync (C:\Code\StyleTest\test-styles\node_modules\sass\sass.dart.js:7650:18)
    at Object.renderSass (C:\Code\StyleTest\test-styles\node_modules\bulma-css-vars\dist\render-sass.js:7:27)
    at C:\Code\StyleTest\test-styles\node_modules\bulma-css-vars\dist\cli.js:90:43
    at Generator.next (<anonymous>)
(node:17312) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:17312) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

//bulma-css-vars.config.js
```
`const { hsl } = require('bulma-css-vars')

// color names have to match bulma's $variable-name, without '$'
// values will be used for initial colors and fallback
const appColors = {
  primary: hsl(161, 49, 58),
  secondary: hsl(355, 52, 48),
  tertiary: hsl(32, 100, 93),
}
// reuse variable colors
appColors['text'] = appColors['primary']

module.exports = {
  sassEntryFile: './src/styles/_main.scss',
  jsOutputFile: './src/bulma-generated/bulma-colors.js',
  sassOutputFile: './src/bulma-generated/generated-vars.sass',
  cssFallbackOutputFile: './src/bulma-generated/generated-fallbacks.css',
  colorDefs: appColors,
  globalWebVar: false,
  transition: '0.5s ease',
}`
```

//_main.scss

```
`// import fallbacks first, so they are overridden
@import '../bulma-generated/generated-fallback.css'; 
@import '../bulma-generated/generated-vars.sass';
@import 'node_modules/bulma-css-vars/bulma-cv-lib';

// Import Bulma's core
@import "node_modules/bulma/sass/utilities/_all";

// I intend to customise Buefy/Bulma here

// Import Bulma and Buefy styles
@import "node_modules/bulma/bulma";
@import "node_modules/buefy/src/scss/buefy";`
```

**Comment by wtho** [comment]
Thanks for reporting! This is definitely something I missed out.

The culprit is [`bulma-css-vars\functions.sass`](https://github.com/wtho/bulma-css-vars/blob/master/lib/functions.sass#L51-L59), which cannnot handle an invocation of `rgba` with four arguments. This has to be adjusted.

**Comment by wtho** [comment]
Should be fixed in `0.5.6`.

Please confirm if your issue was fixed.

Cheers and happy coding!

**Comment by chrishumphrey1985** [comment]
Thanks for sorting. It looks to have worked, but now tackling a sass issue (not resolving node_module references in the file above).

**Comment by wtho** [comment]
It might be due to incompatibility to Bulma v0.8, but I am not sure, I did not double check the changes in Bulma v0.9.

I would recommend to upgrade to Bulma v0.9 to make sure it plays well together.

**Comment by chrishumphrey1985** [comment]
All working now, thanks for your help. 
- Fixed by upgrading Bulma to 0.9.1, Buefy to  0.9.4
- Fixed reference to node_modules, (Did not work using tilde, i.e. ~bulma/bulma).
- Removed extension from import of generated-fallbacks. 

For future reference here's the final scss file: 
```
// import fallbacks first, so they are overridden
@import '../bulma-generated/generated-fallbacks';
@import '../bulma-generated/generated-vars';

@import '../../node_modules/bulma-css-vars/bulma-cv-lib';

// Import Bulma's core
@import "../../node_modules/bulma/sass/utilities/_all";

// Customise Buefy/Bulma here
//....... omitted for brevity

// Import Bulma and Buefy styles
@import "../../node_modules/bulma/bulma";
@import "../../node_modules/buefy/src/scss/buefy";
```

**Comment by wtho** [comment]
Cool, always glad to help!

And yeah, native css variable support in bulma would be the best, but the author seems to be not very keen to dive into it and add it to the library.

Let's hope this will change in the future.
